### ISSUE TITLE
TLS 1.3 only host: TLS 1.2 has a medium finding

### ISSUE THREAD

**Issue by drwetter** [drwetter/testssl.sh#916]
see example in #915. In JSON output:

```{
              "id"           : "tls1_2",
              "ip"           : "tls13.crypto.mozilla.org/52.32.149.186",
              "port"         : "443",
              "severity"     : "MEDIUM",
              "finding"      : "TLSv1.2 is not offered"
          }
,         {
              "id"           : "tls1_3",
              "ip"           : "tls13.crypto.mozilla.org/52.32.149.186",
              "port"         : "443",
              "severity"     : "OK",
              "finding"      : "TLSv1.3 offered: draft 22"
          }
```

``MEDIUM`` seems wrong here.

**Comment by drwetter** [comment]
that was closed via 7ec3c6ab997b012b1518903137d84043cf9825a8. See also #1329

**Comment by drwetter** [comment]
Fee free to use ``~/utils/docker-debian10.tls13only.start.sh``
